src/main/java/com/jmb/springfactory/model/dto/TaskDto.java

```diff
@@ -0,0 +1,55 @@
+package com.jmb.springfactory.model.dto;
+
+import java.util.Date;
+import java.util.List;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class TaskDto extends BaseDto {
+
+  private static final long serialVersionUID = -3407898622118575169L;
+
+  @NotNull
+  private String name;
+
+  @NotNull
+  @Max(value = 255, message = "El campo descripcion tiene un limite de 255 caracteres")
+  private String description;
+
+  @NotNull
+  private Double estimatedTime;
+
+  private String status;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "status" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)